Package.swift

```diff
@@ -32,14 +32,19 @@ let package = Package(
       url: "https://github.com/firebase/nanopb.git",
       "2.30907.0" ..< "2.30908.0"
     ),
+    .package(
+      name: "GoogleUtilities",
+      url: "https://github.com/google/GoogleUtilities.git",
+      "7.2.1" ..< "8.0.0"
+    ),
   ],
   // TODO: Restructure directory structure to simplify the excludes here.
   targets: [
     .target(
       name: "GoogleDataTransport",
       dependencies: [
         .product(name: "nanopb", package: "nanopb"),
-        "GoogleUtilities_Environment",
+        .product(name: "GULEnvironment", package: "GoogleUtilities"),
```

Comment:
How did this build before?!

Reply:
It didn't (at least in this repo) 😄 . `GoogleUtilities_Environment ` dependency was added to the `gdt-promises-main` branch when it was in the main Firebase repo.